sdks/python/apache_beam/io/fileio.py

```diff
@@ -522,7 +522,7 @@ class WriteToFiles(beam.PTransform):
   # Too many files will add memory pressure to the worker, so we let it be 20.
   MAX_NUM_WRITERS_PER_BUNDLE = 20
 
-  DEFAULT_SHARDING = 5
+  DEFAULT_SHARDING = 1
```

Comment:
Is this change necessary? I think line 570 can just check if the input `shards` is None instead of referring to `self.shards > 1`